app/serializers/public_room_serializer.rb

```diff
@@ -21,6 +22,10 @@ def owner_name
     object.user.name
   end
 
+  def owner_id
+    object.user.id
```

Comment:
Don't we wanna at least return the owner id when the request is authenticated and maybe further check the owner role to be sharable?
This can be used to scrap out our rooms owners ids that may empower an attack,
However, this is not a vulnerability.